source/frontend/src/components/common/mapFSM/machineDefinition/mapMachine.ts

```diff
@@ -258,7 +258,6 @@ const selectedFeatureLoaderStates = {
           actions: [
             assign({
               isLoading: () => true,
-              mapLocationSelected: () => null,
```

Comment:
clicking on the map shouldn't hide the marker, users need to see map information around their marker without losing it.